app/Helpers/Helper.php

```diff
@@ -1529,4 +1531,14 @@ static public function getRedirectOption($request, $id, $table, $item_id = null)
         }
         return redirect()->back()->with('error', trans('admin/hardware/message.checkout.error'));
     }
+    public static function unReassignableCount($license)
+    {
+
+        if (!$license->reassignable) {
+            $count = LicenseSeat::where('unreassignable_seat', '=', true)
+                ->where('license_id', '=', $license->id)
+                ->count();
+            return $count;
+        }
+    }
```

Comment:
This method feels like it would be more at home on the actual License model. Is there any reason it needs to be a helper?

Reply:
I may have added at when I was dealing with the transformer. everything there seemed to be coming from the helper file. Moved to the LicenseSeat model 👍